cmd/auth/describe.go

```diff
@@ -0,0 +1,173 @@
+package auth
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/flags"
+	"github.com/databricks/databricks-sdk-go/config"
+	"github.com/spf13/cobra"
+)
+
+var authTemplate = fmt.Sprintf(`{{"Workspace:" | bold}} {{.Details.Host}}.
+{{if .Username }}{{"User:" | bold}} {{.Username}}.{{end}}
```

Comment:
No need for the trailing periods. Double-click to copy/paste can include them erroneously.